src/appleseed/foundation/image/colorspace.h

```diff
@@ -171,6 +173,23 @@ class LightingConditions
 };
 
 
+//
+// Color matching functions and its normalization factor.
+// Used in spectral illuminance conversion.
+//
+
+class ColorMatchingFunction
```

Comment:
It should be `ColorMatchingFunctions` (plural). Comments should also use the plural form.